src/ludamus/templates/chronology/enroll_select.html

```
@@ -1,20 +1,53 @@
+            <span class="tabular-nums">{{ session.agenda_item.start_time|date:"G:i" }}–{{ session.agenda_item.end_time|date:"G:i" }}</span>
+        </span>
+        {% if session.agenda_item.space %}
+            <span class="inline-flex items-center gap-1.5">
+                {% icon "map-pin" class="w-4 h-4" variant="mini" %}
+                {{ session.agenda_item.space.name }}
+            </span>
+        {% endif %}
+        <span class="inline-flex items-center gap-1.5 tabular-nums">
+            {% icon "users" class="w-4 h-4" variant="mini" %}
+            {{ session.enrolled_count }}/{{ session.effective_participants_limit }}
+            {% if session.waiting_count > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)